bears/scss/SCSSLintBear.py

```diff
@@ -1,5 +1,22 @@
+import yaml
+
 from coalib.bearlib.abstractions.Linter import linter
 from coalib.bears.requirements.GemRequirement import GemRequirement
+from coalib.bears.requirements.PipRequirement import PipRequirement
+
+
+def map_naming_convention_to_scsslint(setting):
+    '''
+    This function converts the string ``setting`` (that refers to a naming
+    convention) into a ``scsslint`` specific naming convention.
+    '''
+    naming_convention_map = {
+            'camel': 'camel_case',
+            'snake': 'snake_case',
+            'hyphen': 'hyphenated_lowercase'
+    }
+    return (setting if setting not in naming_convention_map.keys()
+            else naming_convention_map[setting])
```

Comment:
instead of the if, use `.get` with a default value, the whole function is unneeded